code/modules/client/client_colour.dm

```diff
@@ -129,22 +134,25 @@
  * on the client_colour datums it currently has.
  */
 /mob/proc/update_client_colour()
-	if(!client)
+	if(!client || !hud_used)
 		return
-	client.color = ""
-	if(!client_colours.len)
+	var/atom/movable/screen/plane_master/game_plane = hud_used.get_plane_master(MUTATE_PLANE(RENDER_PLANE_GAME, src))
+	game_plane.color = ""
```

Comment:
oh also uh, would prefer this use color filters rather then the color var
both because they're more controllable in terms of application order, but also because any change to alpha would break this color set, and I do that in debugging quite a lot. 

if that's not sensible to you then it's all good, but I prefer it generally